RoboticonColony/Assets/Code/Roboticon.cs

```diff
@@ -38,30 +38,35 @@ public Roboticon(Tile selectedTile)
         bonusProduction[ItemType.Power] = 1;
     }
 
-   
+    
 
     /// <summary>
-    /// Adds the selectd customisation of type RoboticonCustomisations to the Roboticon.
-    /// Adds selected customisation's resource multiplier to the selected bonus 
+    /// Returns the Bonus production of the selected resource
     /// </summary>
-    /// <param name="selectedCustomisation"> An instanciated class of RoboticonCustomisation</param>
-    public void Customise(RoboticonCustomisation selectedCustomisation)
+    /// <param name="resourceType"> The resource type selected (enum ItemType) </param>
+    /// <returns></returns>
+    public int GetBonusProduction(ItemType resourceType)
     {
-        bonusProduction[selectedCustomisation.ResourceType] = selectedCustomisation.BonusMultiplier;
-        CurrentCustomisations.Add(selectedCustomisation);
+        if (resourceType == ItemType.Roboticon){
+            throw new ArgumentException("Roboticon not valid production type");
+        }
+        return bonusProduction[resourceType];
     }
 
     /// <summary>
-    /// Returns the Bonus production of the selected resource
+    /// Sets the bonus multiplier for the selected resource type

```

Comment:
I dont think you should be able to set a bonus production. You should be able  to see the production of the production of a roboticon with maybe a .Produce() method which should throw an error if the roboticon is not on a tile and otherwise calculate production based on the tile and the customisations. Storing production and setting it isnt as good as an idea because people could set it wrong or get it and do wrong calculations with it. If we have just the produce method theres no chance of people getting values and then doing the wrong thing with them.
Sorry for not mentioning this last night i didnt think about it when i was tired haha.